packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -1470,7 +1468,6 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call
     } else if ([@"dispose" isEqualToString:call.method]) {
       [_registry unregisterTexture:cameraId];
       [_camera close];
-      _dispatchQueue = nil;
```

Comment:
If you've never going to nil it out, then it can be created in `init` instead of lazily as it currently is.